prover/src/constraints/evaluation_table.rs

```diff
@@ -164,11 +164,6 @@ impl<'a, E: FieldElement> ConstraintEvaluationTable<'a, E> {
         // by the evaluations of its corresponding divisor, and add all resulting evaluations
         // together into a single vector
         for (column, divisor) in self.evaluations.into_iter().zip(self.divisors.iter()) {
-            // in debug mode, make sure post-division degree of each column matches the expected
-            // degree
-            #[cfg(debug_assertions)]
-            validate_column_degree(&column, divisor, self.domain, column.len() - 1)?;
```

Comment:
is this check now meaningless?

Reply:
When we had the degree adjustments we were always assured that the expected degree is `column.len() -1`. Now that we have removed those we need to computed the expected degree using the highest evaluation degree of the constraints in a group.